source/usage-examples/findOne.txt

```diff
@@ -0,0 +1,53 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+You can retrieve a single document in a collection by chaining together
+the ``find()`` and ``first()`` methods on a ``MongoCollection`` object.
+You can pass a query filter to the ``find()`` method to query for and
+return documents that match the filter in the collection. If you do not
+include a filter, MongoDB returns all the documents in the collection. The
+``first()`` method returns the first matching document.
+
+For more information on querying MongoDB with the Java driver, see our
+guide (TODO: link) to querying documents.
+
+You can also chain other methods to the ``find()`` method
+such as ``sort()`` (TODO: link) which organizes the matched documents in a
+specified order, and ``projection()`` (TODO: link) which configures the
+fields included in the returned documents.
+
+The ``find()`` method returns an object that implements Iterable, an interface
```

Comment:
@jyemin Hope this description makes sense. I welcome any suggestions in improving it.